sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -220,11 +253,19 @@ TrackerT invokeNewTracker(RestrictionT restriction) {
         "Signature is for class %s, but fn is of class %s",
         signature.fnClass(),
         fn.getClass());
+
     try {
       @SuppressWarnings("unchecked")
-      DoFnInvoker<InputT, OutputT> invoker =
-          (DoFnInvoker<InputT, OutputT>)
+      DoFnInvokerBase<InputT, OutputT, DoFn<InputT, OutputT>> invoker;
```

Comment:
`google-java-format` says no (I just autoformat, but try to limit it to lines I touch)